src/shogun/features/CombinedDotFeatures.cpp

```diff
@@ -45,17 +45,17 @@ CCombinedDotFeatures::~CCombinedDotFeatures()
 void CCombinedDotFeatures::list_feature_objs()
```

Comment:
this method here naturally can also be removed ...

Reply:
done